spacy_transformers/pipeline_component.py

```diff
@@ -27,7 +27,7 @@
 @annotation_setters = "spacy-transformers.null_annotation_setter.v1"
 
 [transformer.model]
-@architectures = "spacy-transformers.TransformerModel.v1"
+@architectures = "spacy-transformers.TransformerModel.v2"
 name = "roberta-base"
 tokenizer_config = {"use_fast": true}
 transformer_config = {"output_attentions": false}
```

Comment:
Ah, the other one is secondary, this would be the main place to have it empty by default.

Reply:
I think my reasoning was to show that this is the default behaviour in the config, and the same behaviour as before. But having it as empty boils down to the same thing upon execution.